src/pages/Register/Register.tsx

```diff
@@ -43,17 +46,45 @@ const Register = () => {
         throw new Error("Passwords do not match. Try again.");
       }
       setPasswordMatchError("");
-      await createUserWithEmailAndPassword(
-        auth,
-        formData.email,
-        formData.password
-      );
+
+      const userCredential: UserCredential =
+        await createUserWithEmailAndPassword(
+          auth,
+          formData.email,
+          formData.password
+        );
+      const uid = userCredential.user.uid;
```

Comment:
If it is okay with you, can you add this to the uuid state in the App please? It just saves us from creating a fix,